helm-chart/images/binderhub/binderhub_config.py

```diff
@@ -84,7 +84,8 @@ def get_value(key, default=None):
     c.BinderHub.build_namespace = os.environ['BUILD_NAMESPACE']
 
 if c.BinderHub.auth_enabled:
-    hub_url = urlparse(c.BinderHub.hub_url)
+    hub_url = get_value('config.BinderHub.hub_url', '')
+    hub_url = urlparse(hub_url)
     c.HubOAuth.hub_host = '{}://{}'.format(hub_url.scheme, hub_url.netloc)
```

Comment:
we can be more consistent about getting information off of the config object instead of going all the way back to the yaml. We have similar conditions for when other optional values are missing:

```suggestion
    if "hub_url" in config.BinderHub:
        hub_url = urlparse(config.BinderHub. hub_url)
        c.HubOAuth.hub_host = '{}://{}'.format(hub_url.scheme, hub_url.netloc)
```

but does this work if hub_url is not specified? What happens if hub_url is empty, does the binder auth still work? If it won't work, we should probably raise here instead of producing a known-invalid value

Reply:
thanks for the review Min! 

> but does this work if hub_url is not specified? What happens if hub_url is empty, does the binder auth still work? If it won't work, we should probably raise here instead of producing a known-invalid value

if hub_url is empty, binder auth doesnt work (but I guess it would still work if hub and binder had the same domain). Btw if hub_url is empty, binder launching wouldnt work neither. That is why this step (https://binderhub.readthedocs.io/en/latest/zero-to-binderhub/setup-binderhub.html#connect-binderhub-and-jupyterhub) is necessary, isnt it?

Reply:
I wanted to accept your suggestion already, because I wont be available the coming week. If there will be more changes required, I will do it when I am back